app/api/data-fetching/routes.ts

```diff
@@ -0,0 +1,51 @@
+import { IAPINotRespondingError } from '#models/api-not-responding';
+import { getCarteProfessionnelleTravauxPublic } from '#models/espace-agent/carte-professionnelle-travaux-publics';
+import { EScope } from '#models/user/rights';
+
+const NotImplementedHandler = () => {
+  throw new Error('Not implemented');
+};
+
+export const APIHandler = {
+  'espace-agent/carte-professionnelle-TP': getCarteProfessionnelleTravauxPublic,
+  'espace-agent/conformite': NotImplementedHandler,
+  'espace-agent/opqibi': NotImplementedHandler,
+  'espace-agent/qualibat': NotImplementedHandler,
+  'espace-agent/qualifelec': NotImplementedHandler,
+  'espace-agent/rcs-mandataires': NotImplementedHandler,
+  'espace-agent/rne/documents': NotImplementedHandler,
+  rne: NotImplementedHandler,
+  association: NotImplementedHandler,
+  'verify-tva': NotImplementedHandler,
+  geo: NotImplementedHandler,
+  stats: NotImplementedHandler,
+} as const;
+
+export type APIPath = keyof typeof APIHandler;
+
+export const APIRoutesScope: Record<APIPath, false | EScope> = {
```

Comment:
Why `false` over `null` ?

Reply:
Or why not use a dedicated scope `EScope.none` for open routes ? It would ensure type coherence

Reply:
Don't have a strong opinion. I think I used `false` because initially the variable was called `APIProtectedRoutes`, and it was making more sense that way. But `null` is also OK.